sdk/core/Azure.Core/src/Shared/NextLinkOperationImplementation.cs

```diff
@@ -204,13 +209,13 @@ public RehydrationToken GetRehydrationToken()
             AssertNotNull(response, nameof(response));
             AssertNotNull(finalStateVia, nameof(finalStateVia));
 
-            var headerSource = GetHeaderSource(requestMethod, startRequestUri, response, null, out var nextRequestUri);
+            var headerSource = GetHeaderSource(requestMethod, startRequestUri, response, null, out var nextRequestUri, out var isNextRequestPolling);
```

Comment:
It would be helpful to add a comment explaining what `isNextRequestPolling` is doing and why.

Reply:
added